llm/server.go

```diff
@@ -218,10 +217,6 @@ func NewLlamaServer(gpus discover.GpuInfoList, model string, ggml *GGML, adapter
 		params = append(params, "--threads", strconv.Itoa(defaultThreads))
 	}
 
-	if !opts.F16KV {
```

Comment:
We might want to pay some tech debt on the API type definition.  We may not want to remove it, but at least note it's now ignored and used.